lib/broken_record/result_aggregator.rb

```diff
@@ -29,7 +31,8 @@ def report_results(klass)
         print '[FAIL]'.red
       end
       print "  (#{duration}s)\n"
-      print all_errors.join if all_errors.any?
+
+      print all_errors[0..result_count-1].join if all_errors.any?
```

Comment:
Let's add a note about the total amount of errors for this model, it will help in understanding if this is a minor error and there is only 5 models with errors or if it's like a gazillion. E.g. (untested): 
```ruby
if all_errors.any?
  print all_errors[0..result_count-1].join
  print "Total errrors: #{all_errors.length}" if all_errors.length > result_count
end
```